startup.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+
+from discord.ext import commands
+
+from cogs.utils.config import *
+
+
+# First time run config
+def wizard():
+    config = {}
+    print('Welcome to AlanAyy\'s Ignite-Ember setup!')
+    print('Please enter the bot\'s token:')
+    config['token'] = input('| ').strip().strip('"')
+
+    print('\nPlease enter the bot\'s command prefix:')
+    config['prefix'] = input('| ').strip()
+
+    input('\nThank you! Let\'s start this up, shall we?')
+    with open('settings/config.json', encoding='utf-8', mode='w') as f:
+        json.dump(config, f, sort_keys=True, indent=4)
+
+
+try:
+    with open('settings/config.json', encoding='utf-8', mode='r') as fp:
+        data = json.load(fp)  # Let's see if we've got a valid settings file
+except IOError:
+    wizard()
+
+# Now that all the setup's finished, we're good to go!
+logging.basicConfig(level=logging.INFO)
+bot = commands.Bot(
+    command_prefix=get_config_value('config', 'prefix'),
+    description='''Ignite-Ember by AlanAyy (https://www.alanayy.com/)'''
```

Comment:
Better for a `contributors.md` file

```suggestion
    description='''Ignite-Ember by Schulich Ignite'''
```